### PR TITLE
Dbt e2e test

### PR DESCRIPTION
**Pull request by OnkarVO7**
### Describe your changes :

Added dbt e2e tests

#
### Type of change :

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ingestion

### CODE REVIEW

**Review by pmbrull — COMMENTED** [review]
Thanks for adding this, it will be a lifesaver. I might be missing something by reading the code, but should we add some API validations to check that the DBT models exist, there are tags, owners, and lineage?

IMO each of the supported features should be validated here as a way for us to have an example of everything

File: ingestion/tests/cli_e2e/test_cli_dbt_base.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            self.assert_for_dbt_ingestion(source_status, sink_status)
+
+        # 3. run tests on dbt ingestion
+        @pytest.mark.order(3)
+        def test_entities(self) -> None:
+            for table_fqn in self.fqn_dbt_tables():
+                table: Table = self.openmetadata.get_by_name(
+                    entity=Table, fqn=table_fqn
+                )
+                data_model = table.dataModel
```
Comment (OnkarVO7) [inline ingestion/tests/cli_e2e/test_cli_dbt_base.py:73]: @pmbrull I'm validating the model creation here

File: ingestion/tests/cli_e2e/test_cli_dbt_base.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+                self.assertTrue(len(data_model.columns) > 0)
+                self.assertIsNotNone(data_model.rawSql)
+                self.assertIsNotNone(data_model.sql)
+                self.assertIsNotNone(data_model.upstream)
+                self.assertIsNotNone(data_model.description)
+                self.assertTrue(len(data_model.tags) > 0)
```
Comment (OnkarVO7) [inline ingestion/tests/cli_e2e/test_cli_dbt_base.py:79]: tags are validated here

**Comment by OnkarVO7** [comment]
Will add the lineage and owner validation

File: ingestion/tests/cli_e2e/test_cli_dbt_base.py (hunk 2 above)
Comment (pmbrull) [reply ingestion/tests/cli_e2e/test_cli_dbt_base.py:79]: cool just saw your message below. Thank you

File: ingestion/tests/cli_e2e/test_cli_dbt_base.py (hunk 1 above)
Comment (pmbrull) [reply ingestion/tests/cli_e2e/test_cli_dbt_base.py:73]: thanks for pointing it out

**Review by pmbrull — APPROVED** [review]
Thanks for adding the changes, LGTM

**Pull request merged**